nbs/05_data.transforms.ipynb

```diff
@@ -4,7 +4,17 @@
    "cell_type": "code",
```

Comment:
<p>Not a fan of adding torch here... just use <code>random.sample</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/fastai/fastai/pull/3362/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>